assets/javascripts/modules/analytics/snowplow.js

```diff
@@ -0,0 +1,82 @@
+/*global snowplow_name_here, guardian */
+define(['lodash/object'], function (_) {
```

Comment:
It's not immediately obvious, but it's preferable to only load the methods you need as `lodash/object` will include *all* object methods in our build. You only seem to need `merge` so you *should* be able to include `lodash/object/merge` and then name it as `merge` instead of `_`